std_msgs/Header header
std_msgs/String mode
std_msgs/Int32  position
std_msgs/Float32 force
